src/Creator/InvoiceForOrderCreator.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Creator;
+
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\InvoicingPlugin\Entity\InvoiceInterface;
+use Sylius\InvoicingPlugin\Generator\InvoiceGeneratorInterface;
+use Sylius\InvoicingPlugin\Repository\InvoiceRepository;
+
+final class InvoiceForOrderCreator implements InvoiceForOrderCreatorInterface
```

Comment:
Is there any possibility we would have an invoice that is not **for order**? 😄 And if not, then maybe just `InvoiceCreator` would be sufficient?